config/main.go

```diff
@@ -75,6 +75,14 @@ func start() error {
 }
 
 func exportConfigCmd() error {
+	// Reset the metrics instance name option, as the default
+	// is set to the current hostname.
+	// Config key copied from metrics.CfgOptionInstanceKey.
+	option, err := GetOption("core/metrics/instance")
+	if err == nil {
+		option.DefaultValue = ""
+	}
+
```

Comment:
Wouldn't it be better to mark that option as sensitive and avoid exporting sensitive values? This seems a bit hackish to fix an option from another package here ...

Reply:
Yes, it is hacky.
This is not about the debug data, this is just for the special command where the portmaster just spits out the json and exits.
I think the hacky-ness is acceptable here.

I just don't want the hostname of the current docker container change everytime I export that to the docks.